fiasko_bro/validator_helpers.py

```diff
@@ -1,11 +1,40 @@
 from functools import wraps
 
 
-def tokenized_validator(token):
+def _general_tokenized_validator(tokens, check_method):
```

Comment:
I think we should place this method in `tokenized_validators` module I've mentioned earlier. We could also name it `run_if_tokens_satisfy_condition` and rename the `check_method` argument to `condition`.

And this method will become a part of our public API in case someone needs a more flexible behavior. Also, this method should be documented too.

If you want to leave out some parts of the feature (e.g., documentation), it's OK, just let me now.